pkg/operator/baremetal_config.go

```diff
@@ -76,6 +85,35 @@ func getBaremetalProvisioningConfig(dc dynamic.Interface, configName string) (Ba
 		glog.Errorf("provisioningOSDownloadURL not found in Baremetal provisioning CR %s", configName)
 		return BaremetalProvisioningConfig{}, err
 	}
+	// If provisioningNetwork is not provided, set it to a default of "managed".
+	provisioningNetwork, foundNetwork, err := unstructured.NestedString(provisioningSpec, "provisioningNetwork")
+	if !foundNetwork {
+		glog.Infof("provisioningNetwork not found in Baremetal provisioning CR %s", configName)
+		provisioningNetwork = provisioningNetworkManaged
```

Comment:
I think we want to be able to support migrating clusters that previously had provisioningDHCPExternal. So, if provisioningNetwork isn't found then copy the value from provisioningDHCPExternal, where true == unmanaged, and false == managed.

Reply:
Agree.